tailwind.js

```diff
@@ -138,72 +159,68 @@ module.exports = {
   },
 
   textSizes: {
-    '1': '.625rem',     // 10px
-    '2': '.75rem',     // 12px
-    '3': '.875rem',    // 14px
-    '4': '1rem',     // 16px
-    '5': '1.125rem',   // 18px
-    '6': '1.25rem',    // 20px
+    '1': '.625rem',   // 10px
+    'base': '1rem',   // 16px
```

Comment:
I think having a `.text-base` class is useful for resetting in responsive situations.